client/src/pages/HomePage/HomePage.js

```diff
@@ -3,6 +3,7 @@
 import React from 'react';
 import { Button, Container, Typography, Box, styled } from '@mui/material';
 import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
+import CardContainer from "../../components/CardContainer";
```

Comment:
Please, import file from components folder

Reply:
@Vladislav2498  please fix this.   all our components are imported from the index file in the component folder. please follow the general project scehma.

Reply:
yes, i fixed it